common/src/main/java/org/apache/gravitino/dto/requests/RoleCreateRequest.java

```diff
@@ -78,9 +79,5 @@ public RoleCreateRequest(
   public void validate() throws IllegalArgumentException {
     Preconditions.checkArgument(
         StringUtils.isNotBlank(name), "\"name\" field is required and cannot be empty");
-
-    Preconditions.checkArgument(
-        securableObjects != null && securableObjects.length != 0,
-        "\"securableObjects\" can't null or empty");
```

Comment:
Can you add a UT and IT about the changes here, to check creating a role with empty/null privileges.

Reply:
Added.